config/locales/idv/fr.yml

```
@@ -96,10 +96,7 @@ fr:
     messages:
       activated_html: Votre identité a été vérifiée. Si vous souhaitez modifier votre
         information vérifiée, veuillez %{link}.
-      activated_link: communiquer avec nous
-      agencies_login: Vous pouvez maintenant continuer à vous connecter à ces agences.
-      agency_login_html: Vous pouvez maintenant accéder et vous connecter à
-        <br><strong>%{sp}</strong>.
+      activated_link: communiquer avec nous <br><strong>%{sp}</strong>.
```

I think the markup here was part of the removed text.

```suggestion
      activated_link: communiquer avec nous
```

---

good catch